CSS/styles.css

```diff
@@ -0,0 +1,778 @@
+  transition: all 400ms linear 0.1ms;
+  padding: 1rem 0.5rem;
+  min-height: 5.5rem;
+  gap: 0.6rem;
+}
+
+.program-info img {
+  width: 2.5rem;
+}
+
+.program-info h3 {
+  color: #ec5242;
+  flex: 2;
+}
+
+.program-info p {
+  align-items: center;
+  font-size: 0.875rem;
+  line-height: 1.2rem;
+  color: #d3d3d3;
+  flex: 4;
+}
+
+#join-btn {
+  font-size: 1rem;
+  color: #fff;
+  background-color: #ec5242;
+  transition: all 400ms linear 0.1ms;
+  padding: 1.5rem;
+  cursor: pointer;
+  border: none;
+}
+
+.featured-speakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem 0.8rem 3rem;
+  gap: 1rem;
+}
+
+.more-speakers {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #5a5a5a;
+  border-radius: 3px;
+  padding: 0.7rem 0;
+  margin-top: 1rem;
+  width: 100%;
+  gap: 0.5rem;
+}
+
+.fa-chevron-down {
+  font-size: 1rem;
+  color: #ec5242;
+}
+
+@media screen and (min-width: 768px) {
+  .hamburger,
+  #join-btn,
+  .more-speakers {
+    display: none;
+  }
+
+  .top-nav,
+  .logo,
+  .campaign,
+  .whole-program-btn,
+  #sponsor,
+  footer {
+    display: block;
+  }
+
+  .top-nav ul {
+    display: flex;
+    justify-content: right;
+    align-items: center;
+    background-color: #3e3e3e;
+    padding: 0 11%;
+    gap: 1.5rem;
+  }
+
+  .fa-brands {
+    color: #d3d3d3;
+  }
+
+  .top-nav a {
+    font-size: 0.8rem;
+    text-decoration: none;
+    line-height: 1.6rem;
+    color: #d3d3d3;
+  }
+
+  .bottom-nav {
+    padding: 0 11%;
+  }
+
+  .logo {
+    width: 10rem;
+    flex: 2;
+  }
+
+  .nav-menu {
+    left: auto;
+    width: auto;
+    height: auto;
+    position: initial;
+    flex-direction: row;
+    background: none;
+    padding: 0;
+    flex: 3;
+  }
+
+  .nav-menu li {
+    border: none;
+    width: auto;
+  }
+
+  a.nav-item {
+    font-size: 1rem;
+    line-height: normal;
+    color: #5a5a5a;
+  }
+
+  .campaign .nav-item {
+    font-size: 0.8rem;
+    font-weight: 500;
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 0.5rem;
+  }
+
+  .headline {
+    background-image: url(../Images/Backgrounds/home-bg-desk.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    padding: 6rem 11%;
+  }
+
+  p.welcome {
+    font-size: 2.5rem;
+  }
+
+  h1 {
+    font-size: 3.9rem;
+  }
+
+  p.intro {
+    font-size: 0.9rem;
+    max-width: 73%;
+  }
+
+  .headline h2 {
+    font-size: 1.9rem;
+    margin: 0.5rem 0;
+  }
+
+  p.venu {
+    font-size: 1.3rem;
+  }
+
+  #main-program {
+    padding: 2.5rem 11%;
+  }
+
+  .main-program h2 {
+    font-size: 1.5rem;
+    color: #d3d3d3;
+  }
+
+  .program-info {
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 0.17rem;
+  }
+
+  .program-info li {
+    flex-direction: column;
+    text-align: center;
+    padding: 1.5rem 0.9rem;
+    height: auto;
+    min-height: 14rem;
+    gap: 1rem;
+    flex: 1;
+  }
+
```

Comment:
- [ ] When one of the cards in the main program section is hovered on it should have a white border. Kindly fix this.
Original Design:
 
![image](https://user-images.githubusercontent.com/74814780/205349874-ea3eb009-f0b9-42e0-a5ca-fa3e2a188956.png)

Reply:
[Add hover effect to main program cards](https://github.com/MNisarAli/1st-Capstone-Conference-page/pull/1/commits/8228676ac4e611d0575615e6e021cf3bff8575fd)